unittests/Basic/FileSystemTest.cpp

```diff
@@ -40,7 +40,7 @@ TEST(FileSystemTest, basic) {
     std::error_code ec;
     llvm::raw_fd_ostream os(tempPath.str(), ec, llvm::sys::fs::F_Text);
     EXPECT_FALSE(ec);
-    os << "Hello, world!\n";
+    os << "Hello, world!";
```

Comment:
This was causing problems with `\r\n`. Rather than `ifdef`ing, I figured it should be fine to just not check the newlines